### DISCUSSION TITLE
Lora in Colab

### DISCUSSION THREAD

**Discussion by TintaViva** [lllyasviel/Fooocus#2349] [Q&A] [answered]
How can I load Loras in Colab?

**Comment by eddyizm** [comment, marked as answer]
You can download them from civitia if you have an account, grab your api key and replace the value in the placeholder below. 
You want to do this After you pull in the repo but before you launch it otherwise you will have issues. 

Below is a sample for one i use in my notebook, however you want to get the correct api download url for the one you want to use and rename the safetensor file appropriately.  Also make sure the lora is SDXL. 

`!wget -q --show-progress -O 'models/lora/beksinski.safetensors'  "https://civitai.com/api/download/models/134483?type=Model&format=SafeTensor&token=<YOUR_API_KEY>"`

**Comment by jaimeastorga2000** [comment]
https://www.youtube.com/watch?v=kD6kD6G_s7M&t=255s

Download the LoRA to your computer, load a Fooocus session on Colab, upload the LoRA to the session storage, move the LoRA to the LoRA folder, and refresh all files on the GUI.
